client-src/elements/chromedash-gate-column.js

```diff
@@ -0,0 +1,255 @@
+import {LitElement, css, html, nothing} from 'lit';
+import './chromedash-activity-log';
+import {showToastMessage, findProcessStage} from './utils.js';
+
+import {SHARED_STYLES} from '../sass/shared-css.js';
+
+
+export const STATE_NAMES = [
+  [7, 'No response'],
+  [1, 'N/a or Ack'],
+  [2, 'Review requested'],
+  [3, 'Review started'],
+  [4, 'Needs work'],
+  [5, 'Approved'],
+  [6, 'Denied'],
+];
+
+
+class ChromedashGateColumn extends LitElement {
+  static get styles() {
+    return [
+      ...SHARED_STYLES,
+      css`
+       #votes-area {
+         margin: var(--content-padding) 0;
+       }
+
+       #votes-area table {
+         border-spacing: var(--content-padding-half) var(--content-padding);;
+       }
+
+       #votes-area th {
+         font-weight: bold;
+       }
+
+       #review-status-area {
+         margin: var(--content-padding-half) 0;
+       }
+
+    `];
+  }
+
+  static get properties() {
+    return {
+      user: {type: Object},
+      feature: {type: Object},
+      stage: {type: Object},
+      gate: {type: Object},
+      process: {type: Object},
+      votes: {type: Array},
+      comments: {type: Array},
+      loading: {type: Boolean},
+      needsSave: {type: Boolean},
+    };
+  }
+
+  constructor() {
+    super();
+    this.user = {};
+    this.feature = {};
+    this.stage = {};
+    this.gate = {};
+    this.process = {};
+    this.votes = [];
+    this.comments = [];
+    this.loading = true;
```

Comment:
```suggestion
    this.loading = false;
```
Maybe false by default?